packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -623,14 +621,14 @@ const managerBehavior = {
    * TODO utility method to turn a callback into non-actual one
    * was type {MintAndReallocate}
    *
-   * @param {MethodContext} context
    * @param {Amount<'nat'>} toMint
    * @param {Amount<'nat'>} fee
    * @param {ZCFSeat} seat
    * @param {...ZCFSeat} otherSeats
    * @returns {void}
    */
-  mintAndReallocate: ({ state }, toMint, fee, seat, ...otherSeats) => {
+  mintAndReallocate(toMint, fee, seat, ...otherSeats) {
+    const { state } = this;
```

Comment:
i wonder if automated refactoring tools are up to this sort of thing

Reply:
I wish. I thought of writing one but there are only a few more to transition and once that's done there shouldn't be any new ones.